vignettes/ebola.Rmd

```diff
@@ -0,0 +1,615 @@
+
+```{r epred-sex, fig.cap="(ref:epred-sex)"}
+epred_draws_sex |>
+  dplyr::mutate(
+    sex = dplyr::case_when(
+      sex == "F" ~ "Female",
+      sex == "M" ~ "Male"
+    )
+  ) |>
+  ggplot(aes(x = .epred, y = sex)) +
+  tidybayes::stat_halfeye() +
+  labs(x = "Posterior expectation of the delay", y = "") +
+  theme_minimal()
+```
+
+Finally, the sex-district stratified model is shown in Figure \@ref(fig:epred-sex).
+Note that these results are questionable, and suggest that further would should be done refining the prior distributions for the model.
+Part of the issue is that some sex-district strata have quite limited data.
+
+```{r}
+epred_draws_sex_district <- obs_prep |>
+  as.data.frame() |>
+  modelr::data_grid(sex, district) |>
+  dplyr::mutate(obs_t = NA, pwindow_upr = NA, swindow_upr = NA) |>
+  add_epred_draws(fit_sex_district, dpar = TRUE)
+```
+
+(ref:epred-sex-district) The fitted posterior expectation of the delay distribution for a model with sex-district stratification.
+
+```{r epred-sex-district, fig.cap="(ref:epred-sex-district)"}
+epred_draws_sex_district |>
+  dplyr::mutate(
+    sex = dplyr::case_when(
+      sex == "F" ~ "Female",
+      sex == "M" ~ "Male"
+    )
+  ) |>
+  ggplot(aes(x = .epred, y = district)) +
+  tidybayes::stat_pointinterval() +
+  facet_grid(. ~ sex) +
+  labs(x = "Posterior expectation of the delay", y = "") +
+  theme_minimal()
+```
+
+## Linear predictor posteriors
+
+The `tidybayes` package also allows users to generate draws of the linear predictors for all distributional parameters using `add_linpred_draws`.
+For example, for the `mu` parameter in the sex-district stratified model:
+
+```{r}
+linpred_draws_sex_district <- obs_prep |>
+  as.data.frame() |>
+  modelr::data_grid(sex, district) |>
+  dplyr::mutate(obs_t = NA, pwindow_upr = NA, swindow_upr = NA) |>
+  add_linpred_draws(fit_sex_district, dpar = TRUE)
+```
+
+(ref:linpred-sex-district) The posterior distribution of the linear predictor of `mu` parameter within the sex-district stratified model.
+
+```{r linpred-sex-district, fig.cap="(ref:linpred-sex-district)"}
+linpred_draws_sex_district |>
+  dplyr::mutate(
+    sex = dplyr::case_when(
+      sex == "F" ~ "Female",
+      sex == "M" ~ "Male"
+    )
+  ) |>
+  ggplot(aes(x = mu, y = district)) +
+  tidybayes::stat_pointinterval() +
+  facet_grid(. ~ sex) +
+  labs(x = "Posterior of the mu linear predictor", y = "") +
+  theme_minimal()
+```
+
+## Delay probability mass functions
+
+Posterior predictions of the delay distribution are an important output of an analysis with the `epidist` package.
+In this section, we demonstrate how to produce either a discrete probability mass function representation, or continuous probability density function representation of the delay distribution.
+
+### Discrete probability mass function
+
+The posterior predictive distribution under no truncation but daily censoring.
+
+No stratification:
+
+```{r}
```

Comment:
these are the first unnamed figure chunks

Reply:
Yes sorry, will fix.

Reply:
https://github.com/epinowcast/epidist/pull/52/commits/56781495283b0f816e2e98c656865eca657214db